app/views/includes/layout.njk

```diff
@@ -24,6 +24,9 @@
     <link rel="stylesheet" href="/css/main.min.css">
 
     <script src="/js/main.bundle.min.js" defer></script>
+    <!-- Global site tag (gtag.js) - Google Analytics -->
+    <script async src="https://www.googletagmanager.com/gtag/js?id=UA-170636360-1"></script>
+    <script>window.dataLayer = window.dataLayer || [];function gtag(){dataLayer.push(arguments);}gtag('js', new Date());gtag('config', 'UA-170636360-1');</script>
```

Comment:
If we're using Google analytics we'll need to make sure we handle cookies properly. We'll need:
- a cookie consent banner (https://github.com/nhsuk/cookie-consent)
- a cookie policy page
- a cookie toggle page to opt out / in of cookies

No cookies must be set by Google analytics until they accept the cookie consent banner.

You can find examples of the above on https://www.nhs.uk/ and https://service-manual.nhs.uk/

Reply:
ok thanks.